mantle/platform/qemu.go

```diff
@@ -1028,9 +1028,8 @@ func (builder *QemuBuilder) Exec() (*QemuInstance, error) {
 	if builder.ConfigFile != "" && !builder.configInjected {
 		if builder.supportsFwCfg() {
 			builder.Append("-fw_cfg", "name=opt/com.coreos/config,file="+builder.ConfigFile)
-		} else if builder.primaryDisk == nil {
-			// Otherwise, we should have handled it in builder.AddPrimaryDisk
-			panic("Ignition specified but no primary disk")
+		} else {
+			builder.Append("-drive", fmt.Sprintf("if=none,id=ignition,format=raw,file=%s,readonly=on", builder.ConfigFile), "-device", "virtio-blk,serial=ignition,drive=ignition")
```

Comment:
Optional: might be worth linking https://github.com/coreos/ignition/pull/905 here.

Reply:
Will add both.

Reply:
Done.